Makefile

```diff
@@ -75,7 +75,11 @@ clean:
 # Only build image with lint-free code. CI will fail on lint errors.
 image: lint
 	@if [ $${SKIP_BUILD:-false} = false ] ; then \
-		podman build -t $(IMAGE_TAG) . ; \
+		cp Dockerfile Dockerfile.local ; \
+		if [ -f Dockerfile.patch ] ; then \
```

Comment:
Doesn't this always evaluate to true?

Reply:
yes.  assuming the file is always there.  Can we be guaranteed it is?

Reply:
If it's checked in we can. If someone removes it CI will complain.

Reply:
If someone removes the file then they should also delete this part of the make target. This `-f` check allows them to forget and leave it in as cruft.